.github/workflows/deployment-preview.yml

```diff
@@ -0,0 +1,80 @@
+# .github/workflows/deployment-preview.yml
+name: Deploy PR previews
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+      # TODO: should this be rerun on new commits?
+
+concurrency: preview-${{ github.ref }}
+
+jobs:
+  # TODO: do we still want to run these on dependabot/non-human branches?
+  deploy-preview-docs:
+    if: github.event.action != 'closed'
+    runs-on: ubuntu-20.04
```

Comment:
Why not latest?

Reply:
I was trying to avoid surprise breakages by pinning to a specific version.

Reply:
I'm not worried about os updates.